src/plugins/data_source_management/public/components/utils.test.ts

```diff
@@ -69,6 +71,25 @@ describe('DataSourceManagement: Utils.ts', () => {
     });
   });
 
+  describe('Handle Fetch Data Source Error', () => {
+    const { toasts } = notificationServiceMock.createStartContract();
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    test('should add warning with error message when error is provided', () => {
+      const error = 'Failed to fetch data';
```

Comment:
```suggestion
      const errorMessage= 'Failed to fetch data';
```

or `const error = new Error("Filed to fetch data")`?

Reply:
Got it

Reply:
sure